sonic-thermalctld/scripts/thermalctld

```diff
@@ -515,9 +524,13 @@ class TemperatureUpdater(logger.Logger):
         low_threshold = NOT_AVAILABLE
         high_critical_threshold = NOT_AVAILABLE
         low_critical_threshold = NOT_AVAILABLE
+        maximum_temperature = NOT_AVAILABLE
+        minimum_temperature = NOT_AVAILABLE
         temperature = try_get(thermal.get_temperature)
```

Comment:
Per 10/21 chassis wrkgrp mtg sync up, currently get_temperature() polling interval is set to 60 s.
This could be too relaxed for some vendor's board(s). Like some operate at 10 s polling for thermal (temperature sensors). Can we have an API for platform to provide this polling interval ?

Reply:
Will again add this as a general infra enhancement PR. This might be applicable to all monitoring processes in pmon.